src/features/amUI/users/UsersList.tsx

```diff
@@ -42,6 +81,9 @@ export const UsersList = () => {
           </Heading>
         }
       >
+        {pageEntrees.length === 0 && searchString && (
+          <div>{t('users_page.user_no_search_result')}</div>
```

Comment:
To inform users with screen-readers  of a update on the page, could we add a live region here to inform them that no matches were found? 